app/models/operating_system.rb

```diff
@@ -28,7 +28,7 @@ class OperatingSystem < ApplicationRecord
     ["linux_debian",    %w[debian]],
     ["linux_esx",       %w[vmnixx86 vmwareesxserver esxserver vmwareesxi]],
     ["linux_solaris",   %w[solaris]],
-    ["linux_oracle",    %w[oracle]],
+    ["linux_oracle",    %w[oracle ol]],
```

Comment:
Just to be clear, is this a substring match or a "starts_with" match?  I'm asking because `ol` seems very generic and could accidetally trip on certain strings.

Reply:
Additionally can you add some test cases for this?

Reply:
That's true, its just a `.include?` https://github.com/ManageIQ/manageiq/blob/master/app/models/operating_system.rb#L82

Reply:
Yeah so a future "FooLinux" will match because it's just `ol` in the middle.  Might be nice to allow regexes in that list, and then we can use .match? if it's a regex.

Reply:
We could also have a pre-parse step in the provider for weird cases like this

Reply:
Added oracle cases to the specs and regex matching